arctic/chunkstore/date_chunker.py

```diff
@@ -5,14 +5,16 @@
 
 
 class DateChunker(Chunker):
-    def to_chunks(self, df, chunk_size):
+    TYPE = 'date'
+
+    def to_chunks(self, df, chunk_size='D', **kwargs):
```

Comment:
I put 'daily' because thats just what most of our data is. The user should really specify one, not just reply on the default, but its possible that a chunker might not need a user specified size, so i didnt want to make it a requirement of the design